zerver/openapi/zulip.yaml

```diff
@@ -4653,20 +4653,20 @@ paths:
                                       type: integer
                                       description: |
                                         The [policy][permission-level] for who can use wildcard mentions in
-                                        large streams.
+                                        large channels.
 
-                                        - 1 = Any user can use wildcard mentions in large streams.
-                                        - 2 = Only members can use wildcard mentions in large streams.
-                                        - 3 = Only [full members][calc-full-member] can use wildcard mentions in large streams.
-                                        - 5 = Only organization administrators can use wildcard mentions in large streams.
-                                        - 6 = Nobody can use wildcard mentions in large streams.
-                                        - 7 = Only organization administrators and moderators can use wildcard mentions in large streams.
+                                        - 1 = Any user can use wildcard mentions in large channels.
+                                        - 2 = Only members can use wildcard mentions in large channels.
+                                        - 3 = Only [full members][calc-full-member] can use wildcard mentions in large channels.
+                                        - 5 = Only organization administrators can use wildcard mentions in large channels.
+                                        - 6 = Nobody can use wildcard mentions in large channels.
+                                        - 7 = Only organization administrators and moderators can use wildcard mentions in large channels.
```

Comment:
I'm glad we'll be removing this style of enum-based labels in favor of the new groups-based system; this documentation is just such a mess.